src/language/language.php

```diff
@@ -0,0 +1,36 @@
+<?hh // strict
+require_once($_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php');
+
+/* HH_IGNORE_ERROR[1002] */
+$lang = null;
+
+async function tr_start(): Awaitable<void> {
+  $config = await Configuration::gen('language');
+  $language = $config->getValue();
+  $document_root = must_have_string(Utils::getSERVER(), 'DOCUMENT_ROOT');
+  if(preg_match('/^\w{0,5}$/',$language) and file_exists($document_root . "/language/lang_".$language.".php")) {
```

Comment:
Can you make the `\w{0,5}` into `\w{2}`? We should stick to ISO 639-1 convention for all our language files, where each language code is exactly two characters.